pyproject.toml

```diff
@@ -35,9 +35,28 @@ classifiers = [
   "Topic :: Scientific/Engineering",
 ]
 dynamic = [
-  "dependencies",
   "version",
 ]
+dependencies = [
+  "cf-units>=2; sys_platform!='emscripten'",
+  "cftime>=1.1",
+  "isodate>=0.6.1",
+  "jinja2>=2.7.3",
+  "lxml>=3.2.1",
+  "netcdf4>=1.6.4",
+  "owslib>=0.8.3",
+  "packaging",
+  "pendulum>=1.2.4",
+  "pygeoif>=0.6",
+  "pyproj>=2.2.1",
+  "pyudunits2; sys_platform=='emscripten'",
```

Comment:
@benjwadams this will use pyudunits2 when installing with micropip on pyodide, but won't install cf-units. I guess this is the best we can do for now. If pyudunits2 becomes as fast as cf-units, we can replace cf-units for all cases.